lib/src/command/add.dart

```diff
@@ -92,6 +92,8 @@ class AddCommand extends PubCommand {
         help: 'Build executables in immediate dependencies.');
     argParser.addOption('directory',
         abbr: 'C', help: 'Run this in the directory <dir>.', valueHelp: 'dir');
+    argParser.addFlag('packages-file',
+        help: 'Generate the legacy ".packages" file');
```

Comment:
Should we specify `negatable: false` ? So we don't have `--no-packages-file` option?

Reply:
Yeah - let's do that